src/router/router.js

```diff
@@ -156,16 +156,20 @@ export const navigate = async function () {
         holder.populate({})
         holder.set('w', '100%')
         holder.set('h', '100%')
+
+        const queryParamsData = {}
+        const queryParamsEntries = [...queryParams.entries()]
```

Comment:
since queryParamsEntries is not used anywhere else... I don't think it necessary to do the [...entries] ?